src/app/pages/storage/volumes/datasets/dataset-form/dataset-form.component.ts

```diff
@@ -124,6 +124,17 @@ export class DatasetFormComponent implements Formconfiguration{
         { label: 'lzjb (legacy, not recommended)', value: 'LZJB' }
       ],
     },
+    {
+      type: 'radio',
+      name: 'share_type',
+      placeholder: T('Share Type'),
+      tooltip: T('Choose the type that matches the type of client\
```

Comment:
This is not clear.  Suggested:
`'Set the type of client that will most often use this dataset.'`